azure-pipelines.yml

```diff
@@ -327,6 +327,25 @@ jobs:
         eq(variables['UPDATE_DEPENDENTS'], 'false')
       )
 
+# Replace the intrument key
+    - powershell: |
+            . ./common.ps1
+            $fileToReplace = "$(System.DefaultWorkingDirectory)/nanoFirmwareFlasher.Library/appsettings.json"
+            $sourceString = "INTRUMENT_KEY"
+            $instrumentKey = "$(InstrumentKey)"
+
+            Find-ReplaceInFile -filePath $fileToReplace -sourceString $sourceString -targetString $instrumentKey
+        displayName: Replace intrument key
+        condition: >-
```

Comment:
conditions needs to be adjusted (I'll take care of this, it's a note for myself)